DEV-5/src/Employees/Junior.java

```diff
@@ -0,0 +1,21 @@
+package Employees;
+
+/**
+ * Junior
+ */
+public class Junior extends Employee {
+
+  private int productivity = 1000;
+  private int salary = 1000;
+  private int junCounter = 0;
+
+  @Override
```

Comment:
no reason to have overridden methods - you can just construct base class and pass these values to it